### PR TITLE
Update about page

### PR DESCRIPTION
**Pull request by andrewseamanco**
Update page with new features:

- button styles
- new background image
- elimination of unnecessary if statement in about.js method

### CODE REVIEW

File: portfolio/src/main/webapp/about.js [hunk 1]
```diff
@@ -6,26 +6,26 @@ let correctAnswer = -1;
...
         }
-        if (chosenAnswer != -1) {
-            if (correctAnswer === chosenAnswer) {
-                alert('This is the lie!');
-            } else {
-                alert('This is the truth!');
-            }
+    }
+    console.log(chosenAnswer);
+    console.log(correctAnswer);
```
Comment (tambchop) [inline portfolio/src/main/webapp/about.js]: Get rid of these =)

File: portfolio/src/main/webapp/about.js [hunk 2]
```diff
@@ -6,26 +6,26 @@ let correctAnswer = -1;
...
-    } else {
-        let chosenAnswer = -1;
-        const chosenRadios = document.querySelectorAll('input[name="selected-answer"]');
-        for (chosenRadio of chosenRadios) {
-            if (chosenRadio.checked) {
-                chosenAnswer = chosenRadio.value;
-            }
+    let chosenAnswer = -1;
+    const advanceGameButton = document.getElementById("advance-game-button");
+    if (advanceGameButton.innerHTML==="Play Two Truths One Lie") {
```
Comment (tambchop) [inline portfolio/src/main/webapp/about.js]: Add spacing on the outside of the triple equals.

**Review by tambchop — APPROVED** [review]

**Review by kzweller — APPROVED** [review]

File: portfolio/src/main/webapp/about.js (hunk 2 above)
Comment (andrewseamanco) [reply portfolio/src/main/webapp/about.js]: Changed

File: portfolio/src/main/webapp/about.js (hunk 1 above)
Comment (andrewseamanco) [reply portfolio/src/main/webapp/about.js]: Removed

**Pull request merged**